charts/grafana/ci/with-dashboard-json-values.yaml

```diff
@@ -0,0 +1,53 @@
+dashboards:
+  my-provider:
+    my-awesome-dashboard:
```

Comment:
Do we really need my-awesome-dashboard?

Reply:
These are from the original helm chart which I did not modify.